.env

```
@@ -0,0 +1,3 @@
+DB_URL="postgres://postgres:postgres@localhost:5432/chirpy?sslmode=disable"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)